src/Cookies.php

```diff
@@ -102,6 +102,10 @@ public function addCookie($cookie, $refUri = null)
         }
 
         if ($cookie instanceof SetCookie) {
+            if ($cookie->getMaxAge() < 0) {
```

Comment:
Actually, according to [RFC 6265](http://www.ietf.org/rfc/rfc6265.txt) we should allow negative number (see also #55 comment).